test/runner/configuration/top_level_test.dart

```diff
@@ -148,6 +148,39 @@ void main() {
     test.shouldExit(1);
   });
 
+  test("disables stack trace chaining with chain_stack_traces: false", () {
+    d
+        .file("dart_test.yaml", JSON.encode({"chain_stack_traces": false}))
+        .create();
+
+    d
+        .file(
+            "test.dart",
+            """
+         import 'dart:async';
+
+         import 'package:test/test.dart';
+
+          void main() {
+            test("failure", () async{
+              await new Future((){});
+              await new Future((){});
+              throw "oh no";
+            });
+          }
+    """)
+        .create();
+
+    var test = runTest(["test.dart"]);
+    test.stdout.expect(containsInOrder([
+      "00:00 +0: failure",
+      "00:00 +0 -1: failure [E]",
```

Comment:
This is technically a race condition—it's unlikely but possible that running this test will take longer than a second. Best to just omit the time for good measure, especially since you're already using `containsInOrder()`.